chain/epoch_manager/src/lib.rs

```diff
@@ -1210,7 +1222,12 @@ mod tests {
                 .iter()
                 .map(|x| (x.0.account_id.clone(), x.1))
                 .collect::<Vec<_>>(),
-            vec![("test2".to_string(), false), ("test4".to_string(), false)]
+            vec![
+                // TODO Bowen do they should be sorted by account_id?
```

Comment:
TODOs should have issue attached to them.
Also @bowenwang1996 for actual question.

Reply:
All this TODOs should be resolved before the PR can be merged.